cli_tools/gce_vm_image_import/util/storage_client.go

```diff
@@ -0,0 +1,94 @@
+//  Copyright 2019 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package gcevmimageimportutil
+
+import (
+	"cloud.google.com/go/storage"
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/gce_vm_image_import/domain"
+	"github.com/prometheus/common/log"
+	"google.golang.org/api/option"
+	storagev1 "google.golang.org/api/storage/v1"
+	htransport "google.golang.org/api/transport/http"
+	"io/ioutil"
+	"net/http"
+)
+
+// StorageClient implements domain.StorageClientInterface. It implements main Storage functions
+// used by image import features.
+type StorageClient struct {
+	storageClient *storage.Client
+	httpClient    domain.HTTPClientInterface
+	ctx           context.Context
+}
+
+// HTTPClient implements domain.HTTPClientInterface which abstracts HTTP functionality used by
+// image import features.
+type HTTPClient struct {
+	httpClient *http.Client
+}
+
+// Get executes HTTP GET request for given URL
+func (hc *HTTPClient) Get(url string) (resp *http.Response, err error) {
+	return hc.httpClient.Get(url)
+}
+
+// NewStorageClient creates a StorageClient
+func NewStorageClient(ctx context.Context, client *storage.Client) *StorageClient {
+	o := []option.ClientOption{option.WithScopes(storagev1.DevstorageReadOnlyScope)}
+	hc, _, err := htransport.NewClient(ctx, o...)
+	if err != nil {
+		log.Fatalf("Cannot create storage HTTP client %v", err.Error())
```

Comment:
Don't use fatal logging outside of main, instead return the error and handle it in the callers.

Reply:
Done